assets/tests/components/__snapshots__/propertiesPanel.test.tsx.snap

```diff
@@ -307,6 +315,20 @@ Array [
                 SMITH #op1
               </td>
             </tr>
+            <tr
+              className="m-properties-list__property--last-login"
+            >
+              <td
+                className="m-properties-list__property-label"
+              >
+                Last Login
+              </td>
+              <td
+                className="m-properties-list__property-value"
+              >
+                4h 3m; 1:38pm
```

Comment:
Adding the logon time to the properties list also adds it to the VPP. I just want to double check that we want that.

Reply:
Yes, good question. I discussed with Ingrid and we are doing that intentionally.